source/base/DateTime.ooc

```diff
@@ -14,22 +14,34 @@
 * You should have received a copy of the GNU Lesser General Public License
 * along with this software. If not, see <http://www.gnu.org/licenses/>.
 */
-TimeSpan: cover {
-	_ticks: Int64
-	ticks ::= this _ticks
-	init: func@ (=_ticks)
-	kean_base_timeSpan_getTicks: unmangled func -> Int64 { this _ticks }
-	new: unmangled(kean_base_timeSpan_new) static func ~API (ticks: UInt64) -> This { This new(ticks) }
-}
+
+import TimeSpan
 
 DateTime: cover {
 	/* Number of 100 ns intervals since 00.00 1/1/1 */
 	_ticks: UInt64
 	ticks ::= this _ticks
-
 	init: func@ (=_ticks)
+	init: func ~fromYearMonthDay (year : Int, month: Int, day: Int) {}
+	init: func ~fromHourMinuteSec (hour : Int, minute : Int, second : Int){
```

Comment:
I'm pretty sure it needs to be `func@` for all the `init` functions in `cover`s.